src/Go/effects/effect.ts

```diff
@@ -133,3 +137,23 @@ export function getDifficultyMultiplier(komi: number, boardSize: number) {
   const isTinyBoardVsIlluminati = boardSize === 5 && komi === opponentDetails[GoOpponent.Illuminati].komi;
   return isTinyBoardVsIlluminati ? 8 : (komi + 0.5) * 0.25;
 }
+
+export function gainFavor(opponent: GoOpponent, statusToUpdate: OpponentStats): void {
```

Comment:
This snuck in from a different branch?

Reply:
Ah, that was some scratch testing from a report where gang factions weren't getting favor reward (though they do, it turns out.) Removed.